scripts/build/Platform/Windows/deploy_cdk_applications.cmd

```diff
@@ -0,0 +1,72 @@
+@ECHO OFF
+REM
+REM Copyright (c) Contributors to the Open 3D Engine Project.
+REM For complete copyright and license terms please see the LICENSE at the root of this distribution.
+REM
+REM SPDX-License-Identifier: Apache-2.0 OR MIT
+REM
+REM
+
+SETLOCAL EnableDelayedExpansion
+
+SET SOURCE_DIRECTORY=%CD%
+SET PATH=%SOURCE_DIRECTORY%\python;%PATH%
+SET GEM_DIRECTORY=%SOURCE_DIRECTORY%\Gems
+
+ECHO [cdk_installation] Install the latest version of CDK

```

Comment:
Why are we not doing this inside a venv? We are impacting the machine running this?

Reply:
We can do this inside a venv. We're using the O3DE python for our current automation tests and I just wanted to keep consistency on that.

Reply:
A straight call to python will invoke the build machine's python. If you want to run O3DE's python, you will need to reference the python\python.cmd wrapper script. But since you are creating a venv here anyways, I guess its safe

Reply:
I've added the o3de python path to the environment variable using:
"SET PATH=%SOURCE_DIRECTORY%\python;%PATH%"

Also created a venv here with the latest change.